lawlibrary/templates/peachjam/layouts/main.html

```diff
@@ -1,10 +1,96 @@
 {% extends "peachjam/layouts/main.html" %}
+{% load i18n %}
 
 {% load sass_tags %}
 
 {% block head-meta %}
 {% endblock %}
 
+{% block head-js %}
+<!--Loads JQuery asynchronously-->
+<script type="application/javascript" src="//s3.amazonaws.com/downloads.mailchimp.com/js/mc-validate.js" defer></script>
+<script type="application/javascript" defer>
+  var jqueryLoaded = new Promise(function (resolve) {
+    var interval = setInterval(function () {
+      if (window.jQuery) {
+        clearInterval(interval);
+        resolve();
+      }
+    }, 200);
+  });
+
+  jqueryLoaded.then(function () {
+    window.fnames = [];
+    window.ftypes = [];
+    fnames[0] = 'EMAIL';
+    ftypes[0] = 'email';
+    var $mcj = window.jQuery.noConflict(true);
+  });
+
+</script>
+{% endblock %}
+
 {% block head-css %}
-  <link rel="stylesheet" href="{% sass_src 'stylesheets/lawlibrary.scss' %}" type="text/css">
+<link rel="stylesheet" href="{% sass_src 'stylesheets/lawlibrary.scss' %}" type="text/css">
+{% endblock %}
+
+{% block body-content %}
+  {{ block.super }}
+    <section class="bg-dark pb-5 pt-5">
+      <div class="container">
+        <div class="row">
+          <div class="col-sm-12 col-lg-6">
+            <h2 class="mb-2 text-white">{% trans 'Subscribe' %}</h2>
+            <p class="lead text-white">
+              {% trans 'Sign up for news from LawLibrary ZA. Complete the form below to be added to our mailing list.' %}
```

Comment:
Rule of thumb: if you find yourself writing instructions like "fill in the form below" or "click the button to the right" then your copy is poor and should be re-worked.

```suggestion
              {% trans 'Subscribe to the LawLibrary newsletter for updates and news on South African legal information.' %}
```